src/languageTools/LanguageClient/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "langaugeclient",
```

typo

```suggestion
  "name": "languageclient",
```

or maybe something like

```suggestion
  "name": "brackets-language-client",
```

---

Fixed